### PR TITLE
added support for counters

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

**Comment by roydahan** [comment]
Next times, please include more details about the commit in the commit message.

As a background for reviewers, the idea was to support another stress command that will be run in parallel to the stress_cmd.
In this case, it's for c-s counters-write command that will be run in longevity in parallel to the regular cmd.

Another support added here, is for experimental flag.
Counters were introduced as experimental in 1.7 and other features may be introduced as such in future.
This way, one will be able to set experimental: true in test.yaml in order to enable these features.

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1660,6 +1660,13 @@ def stop_nemesis(self, timeout=10):
         self.nemesis_threads = []
         self.log.debug('Stop nemesis end')
 
+    def _experimental(self):
+        for cmd in ('stress_cmd', 'stress_cmd_1'):
+            stress_cmd = self.params.get(cmd)
+            if stress_cmd and 'counter_' in stress_cmd:
+                return True
+        return False
+
```
Comment (roydahan) [inline sdcm/cluster.py]: Rethinking about it, I think we said we don't need this check.
This is relevant only to 1.7 and we said that we can explicitly mention it in the yaml file.

Comment (amoskong) [inline sdcm/cluster.py]: So  we need to set experimental to true in setup scylla

File: longevity_test.py [hunk 2]
```diff
@@ -36,10 +36,55 @@ def test_custom_time(self):
...
         self.db_cluster.wait_total_space_used_per_node()
         self.db_cluster.start_nemesis(interval=self.params.get('nemesis_interval'))
-        self.verify_stress_thread(queue=stress_queue)
+
+        for stress in stress_queue:
+            self.verify_stress_thread(queue=stress)
+
+    def _create_counter_table(self):
+        """
+        workaround for the issue #32
```
Comment (amoskong) [inline longevity_test.py]: How about add the full url of issue? What's the issue #32?

Comment (larisau) [inline longevity_test.py]: added

File: sdcm/cluster.py (hunk 1 above)
Comment (larisau) [inline sdcm/cluster.py]: changed, we just need to add the following line to tests yaml:
experimental: 'true'

**Pull request merged**